Functions.Templates/Templates/BlobTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -51,7 +51,7 @@
             "ManualInstructions": [],
```

Comment:
Can you also update the description above to match the new pkg we are using:

`"Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs Nuget package",`